client/me/account-password/index.jsx

```diff
@@ -58,13 +58,23 @@ class AccountPassword extends Component {
 			return;
 		}
 
-		try {
-			const validation = await wp.req.post( '/me/settings/password/validate', { password } );
-
-			this.setState( { pendingValidation: false, validation } );
-		} catch ( err ) {
-			this.setState( { pendingValidation: false } );
-		}
+		const validation = await wp.req
+			.post( '/me/settings/password/validate', { password } )
+			.catch( () => ( {
+				passed: false,
+				test_results: {
+					failed: [
+						{
+							test_name: 'network_error',
+							explanation: this.props.translate(
+								'The password could not be vaiidated. Please check your network connection and try again.'
```

Comment:
There's a typo, it should be `validated`.

Reply:
Oh that's a great catch, thanks! Fixed.